presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -309,4 +317,20 @@ private static String randomNameSuffix()
         String randomSuffix = Long.toString(abs(random.nextLong()), MAX_RADIX);
         return randomSuffix.substring(0, min(RANDOM_SUFFIX_LENGTH, randomSuffix.length()));
     }
+
+    @Test
+    public void testInsertForDefaultColumn()
+            throws SQLException
+    {
+        String tableName = "table_" + randomNameSuffix();
```

Comment:
Best would be to use `TestTable`. Alternatively you can use something like `io.prestosql.plugin.mysql.TestMySqlCaseInsensitiveMapping#withTable`